polkadot/runtime/parachains/src/mock.rs

```diff
@@ -388,6 +385,101 @@ impl ValidatorSetWithIdentification<AccountId> for MockValidatorSet {
 	type IdentificationOf = FoolIdentificationOf;
 }
 
+pub mod mock_assigner {
+	pub use pallet::*;
+	use super::*;
+	#[frame_support::pallet]
+	pub mod pallet {
+		use super::*;
+
+		#[pallet::pallet]
+		#[pallet::without_storage_info]
+		pub struct Pallet<T>(_);
+
+		#[pallet::config]
+		pub trait Config: frame_system::Config + configuration::Config + paras::Config {}
+
+		/// Creates an empty on demand queue if one isn't present in storage already.
+		#[pallet::type_value]
+		pub(super) fn OnDemandQueueOnEmpty<T: Config>() -> VecDeque<V0Assignment> {
```

Comment:
Why do we need this in mocking?

Reply:
The tests passed without it, so it seems not.